lib/bokken_web/controllers/file_controller.ex

```diff
@@ -11,7 +17,49 @@ defmodule BokkenWeb.FileController do
     end
   end
 
-  def images(conn, %{"type" => type, "id" => id, "file" => image}) do
+  def files(conn, %{"type" => type, "id" => id, "file" => image})
+      when is_image(type) or type in ~w(projects) do
     send_file(conn, 200, "uploads/#{type}/#{id}/#{image}")
```

Comment:
the path to the files is uploads/type/version/id/file.
Here you don't have the version

Reply:
In the case it is a snippet, the type is snippet and therefor the version.